requirements.txt

```diff
@@ -26,9 +26,9 @@ pycparser==2.21
 python-constraint==1.4.0
 python-dateutil==2.8.2
 qiskit==0.42.1
-qiskit-aer==0.10.1
+qiskit-aer==0.12.0
 qiskit-ibm-runtime==0.9.3
```

Comment:
We can probably bump qiskit and provider.

Reply:
already at latest